Api/Api.cs

```diff
@@ -384,9 +384,15 @@ public Backtest CreateBacktest(int projectId, string compileId, string backtestN
                 backtestName
             }), ParameterType.RequestBody);
 
-            Backtest result;
+            BacktestResponseWrapper result;
             ApiConnection.TryRequest(request, out result);
-            return result;
+
+            // Use API Response values for Backtest Values
+            result.Backtest.Success = result.Success;
+            result.Backtest.Errors = result.Errors;
+
+            // Return only the backtest object
+            return result.Backtest;
```

Comment:
Do we **need** this wrapper? Looks fishy, incompatibility between Lean and cloud api

Reply:
With the new response structure it is, we could remove the success and error members from the backtest class though and not assign them like it is now. I wanted to keep those members though for users to know what could've gone wrong in their api call, because I stuck with returning just the backtest obj instead of returning the wrapped obj in the new class that contains those details